bundles/core/src/main/java/com/adobe/cq/wcm/core/components/config/PageItemsConfig.java

```diff
@@ -0,0 +1,61 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2020 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.wcm.core.components.config;
+
+import org.apache.sling.caconfig.annotation.Configuration;
+import org.apache.sling.caconfig.annotation.Property;
+
+/**
+ * Context-aware configuration holding information on items to be included in pages:
+ *      - scripts
+ *      - links
+ *      - meta
+ *  This configuration is meant to be used as a context-aware resource.
+ *  See <a href="https://sling.apache.org/documentation/bundles/context-aware-configuration/context-aware-configuration.html#context-aware-resources">Context-Aware Resources</a>
+ *
+ *  Assumed structure:
+ *
+ *  <pre>
+ *      com.adobe.cq.wcm.core.components.config.PageItemsConfig
+ *          + prefixPath="/some/path"
+ *          - item01
+ *              + element=["link"|"script"|"meta"]
+ *              + location=["header"|"footer"]
+ *              + attributeName="attributeValue"
+ *          - item02
+ *              ...
+ *  </pre>
+ */
+@Configuration(label = "Page Items", description = "Context-Aware Configuration for items that will be included in the page")
+public @interface PageItemsConfig {
+
+    /**
+     * Name of the property that store the prefix path
+     *
+     * @since com.adobe.cq.wcm.core.components.config 1.0.0
+     */
+    String PROP_PREFIX_PATH = "prefixPath";
```

Comment:
I would not introduce a path prefix. The property that stores the HTML attribute (`src` or `href`) should have the complete path or URL.

Reply:
The path prefix allows easier updating of all paths, when using CI/CD driven UI artifacts deployment, eg.: from `/static/<build1>/....` to `/static/<build2>/...`. One can leave it empty, if they don't need or want to use it.

Reply:
I find this property confusing. I would at least document that this path is prepended to the `src` and `href` properties when those attributes are set.

Reply:
That's documented at https://github.com/adobe/aem-core-wcm-components/pull/1203/files#diff-8d29d2fab8e7f9b29565e6404749382cR56 (3 lines down) ;)